packages/core/src/App/Containers/PasswordSelectionModal/password-selection-modal.jsx

```diff
@@ -88,17 +88,19 @@ const PasswordSelectionModal = ({
                 </div>
             </React.Fragment>
         ) : (
-            <Button
-                className={classNames('account-signup__btn', {
-                    'account-signup__btn--disabled': !values.password || errors.password || isSubmitting,
-                })}
-                id='dt_core_account-signup-modal_submit-btn'
-                type='submit'
-                is_disabled={!values.password || !!errors.password || isSubmitting}
-                text={localize('Start trading')}
-                large
-                primary
-            />
+            <div className='account-signup__footer'>
```

Comment:
Why do you need this change?

Reply:
The Button was not properly align. And in the base style for the button it shows that `align-self: flex-end;` cannot be applied properly hence why I added a container `'account-signup__footer'` and set the style for the container:
```
display: flex;
justify content: flex-end
```
Just to make sure that the button in Desktop is align properly as per design.